### PR TITLE
Added documentation to bookmarklet

### PR DESCRIPTION
**Pull request by cdelahousse**
@spratt was confused and scared. This should alleviate that.

### CODE REVIEW

File: src/bookmarklet/Makefile [hunk 1]
```diff
@@ -1,17 +1,72 @@
+# Copyright by Steamed Pears, 2013. For licensing information, 
+# see the LICENCE file in the root directory of this project.
```
Comment (spratt) [inline src/bookmarklet/Makefile:2]: Nice.

File: src/bookmarklet/Makefile [hunk 2]
```diff
@@ -1,17 +1,72 @@
...
 
+# Root folder where the bookmarklet will be built. Typically, this would be the
+# folder that would be pushed to production.
 TARGET=../client/bookmarklet
+
+# Where raw dependencies/vendor code are stored. This folder is populated by the
+# get get vendor target
```
Comment (spratt) [inline src/bookmarklet/Makefile]: get get?

File: src/bookmarklet/README.md [hunk 3]
```diff
@@ -1,26 +1,31 @@
 #Code Review Bookmarklet
 
-##Development Usage
+##Usage
 
-Run `make install` to get the bookmarklet to a working state. It will download all
-the vendor dependencies, build them, and move everything to the static file
-server.
+Open `index.html` and drag one of the bookmarklet links to you bookmarks bar.
+Boom!  You can now use it.
```
Comment (spratt) [inline src/bookmarklet/README.md:6]: Could the build process also install an html file to the src/client/ directory?  Maybe we should put a link to the bookmarklet on the main UI?  

The users of the app need a clear way to use the bookmarklet.

File: src/bookmarklet/README.md [hunk 4]
```diff
@@ -1,26 +1,31 @@
...
-Run `make build` to build the bookmarklet and vendors and move them to the
-static file server.
+##Development
+
+After cloning the project, run `make install` to get the bookmarklet to a
+working state. It will download all the vendor dependencies, build them, and
+move everything to them to where they can be served.
+
+Run `make build` to build the test-build version of the bookmarklet and vendors
+and move them to where files are are served.
```
Comment (spratt) [inline src/bookmarklet/README.md:15]: This is really clear.

[goodjob]

File: src/bookmarklet/README.md (hunk 3 above)
Comment (spratt) [inline src/bookmarklet/README.md:6]: I think this is [blocking] because this is a large part of my concern.

Comment (cdelahousse) [inline src/bookmarklet/README.md:6]: No. Index.html is just an example for developers.

We should copy one of the links to the homepage.

Comment (spratt) [inline src/bookmarklet/README.md:6]: Right, so we should put a link to the bookmarklet in the main UI?

Comment (cdelahousse) [inline src/bookmarklet/README.md:6]: Yes. It'll be my next commit.

Comment (spratt) [inline src/bookmarklet/README.md:6]: Cool.

Comment (cdelahousse) [inline src/bookmarklet/README.md:6]: 7b1c873ddcf1f1f129860fe1c54049e7c8b08264

**Pull request merged**